.github/workflows/integration-tests.yml.in

```diff
@@ -446,12 +470,12 @@ jobs:
       - name: Install brew dependencies
         run: |
           brew update
-          brew install ccache llvm@19 lld
+          brew install ccache llvm@19 lld coreutils
```

Comment:
Which tool from `coreutils` are you using here?

Reply:
https://github.com/triton-lang/triton/pull/5202#discussion_r1851019895